codegen/build.sbt

```diff
@@ -2,24 +2,24 @@ import com.nitro.build._
 
 import PublishHelpers._
 
+ScriptedPlugin.scriptedSettings
```

Comment:
I don't know about this -- what does this do?

Reply:
I found this one in the following issue description. Had similar issues when I first tried to compile it with another version of sbt:
https://github.com/sbt/sbt/issues/3325